cabal-install/Distribution/Client/Dependency/Types.hs

```diff
@@ -1,16 +1,4 @@
 {-# LANGUAGE DeriveGeneric #-}
------------------------------------------------------------------------------
--- |
--- Module      :  Distribution.Client.Dependency.Types
--- Copyright   :  (c) Duncan Coutts 2008
--- License     :  BSD-like
---
--- Maintainer  :  cabal-devel@haskell.org
--- Stability   :  provisional
--- Portability :  portable
---
--- Common types for dependency resolution.
```

Comment:
I actually removed it because the "purpose" isn't very clear any longer. The module is now a sort of weird mish-mash.

(I think this module should perhaps also be split, but I'm kind of waiting to see where migrating the Tests takes me... The module may become entirely redundant.)